.github/ISSUE_TEMPLATE/4_e2e_acceptance_error_template.yml

```diff
@@ -0,0 +1,110 @@
+name: E2E or Acceptance Test CI Failure Report
```

Comment:
One other suggestion -- let's make this the 3rd template in the list, i.e. put it above the existing CI error template. I think this one will be more common.